crates/ruff_python_formatter/src/other/parameters.rs

```diff
@@ -202,14 +243,22 @@ impl FormatNodeRule<Parameters> for FormatParameters {
             // No parameters, format any dangling comments between `()`
             write!(
                 f,
-                [
+                [empty_parenthesized_with_dangling_comments(
                     text("("),
-                    block_indent(&dangling_comments(dangling)),
-                    text(")")
-                ]
+                    dangling,
+                    text(")"),
+                )]
             )
         } else {
-            write!(f, [parenthesized("(", &group(&format_inner), ")")])
+            write!(
+                f,
+                [parenthesized_with_dangling_comments(
```

Comment:
Now using the same utilities as all the other nodes (`parenthesized_with_dangling_comments`, `empty_parenthesized_with_dangling_comments`).